dev/NavigationView/NavigationView.xaml

```diff
@@ -541,6 +549,7 @@
                                             <!-- FooterItems -->
                                             <local:ItemsRepeaterScrollHost Grid.Row="3">
                                                 <ScrollViewer x:Name="FooterItemsScrollViewer"
+                                                        VerticalScrollBarVisibility="Auto"
```

Comment:
If not set to atuo, verticalScrollBar will always be visible and will take space even when there are no items on the footer

Reply:
There is already a PR for this too: #3777

Reply:
@chingucoding could you please add the scrollbar change to your PR? I will close #3777

Reply:
Added the changes from this PR and #3777 now.

Reply:
awesome. I will verify your changes in a bit :)